common/tests/threadpool_int/threadpool_int.c

```diff
@@ -143,6 +152,44 @@ static void wait_work_function(void* context)
     wake_by_address_single_64(&wait_work_context->call_count);
 }
 
+static size_t get_work_per_thread()
+{
+    size_t work_per_thread = WORK_PER_THREAD;
+#ifdef USE_VALGRIND
+    // Check if we are being invoked from Valgrind to ensure that
+    // the non-Valgrind and Helgrind run do not Yield
+    if (RUNNING_ON_VALGRIND)
+    {
+        // Yield
+        work_per_thread = WORK_PER_THREAD / 10;
+    }
+#endif
+    return work_per_thread;
+}
+
+static int schedule_work_multiple_threads(void* context)
```

Comment:
>schedule_work_multiple_threads [](http://example.com/codeflow?start=11&length=30)

I don't understand what does this function have to do with multiple threads.
It is called from one thread (the main thread that executes the test).
It schedules a number of items to execute.
So where is the multiple threads?